src/System.Net.Http/dir.props

```diff
@@ -6,5 +6,6 @@
     <AssemblyKey>MSFT</AssemblyKey>
     <IsNETCoreApp>true</IsNETCoreApp>
     <IsUAP>true</IsUAP>
+    <IsNetFxNETStandard>true</IsNetFxNETStandard>
```

Comment:
For assemblies like System.Net.Http that still ship an individual package do we want to keep the netfx asset in that package as well and let conflict resolution deal with them? Or should we put placeholders in stead.